src/main/java/core/basesyntax/operation/Balance.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.operation;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.db.StorageImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationService;
+
+public class Balance implements OperationService {
+    private StorageImpl storageImpl;
+
+    public Balance(StorageImpl storage) {
+        this.storageImpl = storageImpl;
+    }
+
+    @Override
+    public void operation(FruitTransaction fruitTransaction) {
```

Comment:
```suggestion
    public void process(FruitTransaction fruitTransaction) {
```
please rename all cases in other classes